### PR TITLE
adding headers to public data output files

### LINKED ISSUE #2979: Add Headers to Dynamic Public LAR Data PSV
- [ ] Append header rows to the dynamic data sets being generated (hmda-regulator/hmda-data-publisher)

### PR DESCRIPTION
**Pull request by PatrickHSI**
closes #2979

### CODE REVIEW

**Comment by PatrickHSI** [comment]
Leaving this as a WIP until david can verify the data files being generated

**Comment by PatrickHSI** [comment]
output confirmed to be correct by @Kibrael

File: hmda-regulator/src/main/resources/application.conf [hunk 1]
```diff
@@ -44,38 +46,36 @@ hmda {
...
 public-aws {
   public-access-key-id = ""
   public-access-key-id = ${?AWS_ACCESS_KEY_ID}
   public-secret-access-key = ""
   public-secret-access-key = ${?AWS_SECRET_ACCESS_KEY}
-  public-environment = "dev" //change to "prod" for production S3 bucket
+  public-environment = "dev"
   public-environment = ${?AWS_ENVIRONMENT}
   public-s3-bucket = "cfpb-hmda-export"
   public-s3-bucket = ${?S3_PRIVATE_BUCKET}
```
Comment (omnipresent07) [inline hmda-regulator/src/main/resources/application.conf]: `public-s3-bucket` should be `cfpb-hmda-public` and env variable should be `S3_PUBLIC_BUCKET`

**Comment by omnipresent07** [comment]
LGTM 👍

**Pull request merged**